src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  font-family: Robotos, sans-serif;
```

Comment:
Consider moving this style to the body selector, to get the font family inherited by other elements